<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sniffer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="FramesTableView" name="tableView_packets">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_infoView">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_hexContent">
          <property name="font">
           <font>
            <family>Fira Code Medium</family>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPlainTextEdit{
border: 1px solid;
border-color: black transparent black black;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="documentTitle">
           <string notr="true"/>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="plainText">
           <string notr="true"/>
          </property>
          <property name="maximumBlockCount">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_asciiContent">
          <property name="font">
           <font>
            <family>Fira Code Medium</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPlainTextEdit{
border: 1px solid;
border-color: black black black transparent;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSniffer">
    <property name="title">
     <string>Sniffer</string>
    </property>
    <addaction name="actionShow_device_list"/>
    <addaction name="actionResolve_address_to_domain_names"/>
    <addaction name="separator"/>
    <addaction name="actionStart_listening"/>
    <addaction name="actionStop_listening"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_dump_file"/>
    <addaction name="actionSave_dump_file"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionClear_all_packets"/>
    <addaction name="actionLog_window"/>
   </widget>
   <addaction name="menuSniffer"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_logger">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="LogWidget" name="logWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>60</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>95</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::WidgetWidth</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximumBlockCount">
        <number>-3</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart_listening"/>
   <addaction name="actionStop_listening"/>
   <addaction name="separator"/>
   <addaction name="actionShow_device_list"/>
  </widget>
  <action name="actionShow_device_list">
   <property name="text">
    <string>Show device list</string>
   </property>
  </action>
  <action name="actionStart_listening">
   <property name="text">
    <string>Start listening</string>
   </property>
  </action>
  <action name="actionStop_listening">
   <property name="text">
    <string>Stop listening</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLog_window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log window</string>
   </property>
  </action>
  <action name="actionClear_all_packets">
   <property name="text">
    <string>Clear all packets</string>
   </property>
  </action>
  <action name="actionSave_dump_file">
   <property name="text">
    <string>Save dump file</string>
   </property>
  </action>
  <action name="actionLoad_dump_file">
   <property name="text">
    <string>Load dump file</string>
   </property>
  </action>
  <action name="actionResolve_address_to_domain_names">
   <property name="text">
    <string>Resolve address to domain names</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>Widgets/LogWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>FramesTableView</class>
   <extends>QTableView</extends>
   <header>Widgets/FramesTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
